reports/2022.html

```diff
@@ -141,9 +141,9 @@ <h3>Table of Contents</h3>
             </tr>
             <tr>
               <th><a href="#themeing">Themeing</a></th>
```

Comment:
A bit late to ask this, but should it be "Themeing" or "Theming"?

Reply:
Yeah, it probably should be [the latter](https://en.wikipedia.org/wiki/Theming).